lib/reversetunnel/transport.go

```diff
@@ -101,13 +101,17 @@ type dialReq struct {
 	// special non-resolvable address like @remote-auth-server.
 	Address string `json:"address,omitempty"`
 
-	// SearchNames is a list of aliases for the host. SearchNames is used when
+	// ServerID is the hostUUID.clusterName of th enode. ServerID is used when
 	// dialing through a tunnel.
-	SearchNames []string `json:"search_names,omitempty"`
+	ServerID string `json:"server_id,omitempty"`
```

Comment:
Won't this change in the json tag here and below cause some sort of compatibility issues b/w older/newer clusters?

Reply:
This feature has not been released officially yet, will be in 4.0.0 final release.